sentinelsat/scripts/cli.py

```diff
@@ -69,7 +80,11 @@ def convert(self, value, param, ctx):
     help="End date of the query in the format YYYYMMDD.",
 )
 @click.option(
-    "--geometry", "-g", type=click.Path(exists=True), help="Search area geometry as GeoJSON file."
+    "--geometry",
+    "-g",
+    type=str,
+    help="Search area geometry as GeoJSON file, a GeoJSON string, or a WKT string. "
+    "Valid WKT objects are Polygon and Point.",
```

Comment:
This note is directly from the Copernicus Open Access Hub documentation. Seems like this is again a point where the documentation deviates from their own implementation.

Let's got with @valgur comment then and drop this limitation. This will bubble-up as an error should the server drop the support for other WKT types.